person.rb

```diff
@@ -1,21 +1,36 @@
-class Person
+require './nameable'
+require './capitalize_decorator'
+require './timmwe_decorator'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] Kindly remove `'unknown'` from the name parameter passed, this will erase the error in the terminal 👍🏿 

![Screenshot_529](https://user-images.githubusercontent.com/69638013/202211529-e264a0ee-f097-4552-a835-e320b5df8ce4.png)

Reply:
@prolajumokeoni  thank you very much I fix it :)